nix/evmdis.nix

```diff
@@ -1,21 +0,0 @@
-{ stdenv, lib, buildGoPackage, fetchFromGitHub }:
```

Comment:
I haven't used evmdis for a while, but we do advertise it on dapp.tools

Reply:
The version w

Reply:
The version we have here is more than 3 years old, I had assumed that it would be broken on most new contracts?

Reply:
There is a maintained and up to date version in `nixpkgs` fwiw: https://github.com/NixOS/nixpkgs/blob/84cf00f98031e93f389f1eb93c4a7374a33cc0a9/pkgs/development/tools/analysis/evmdis/default.nix